src/config.rs

```diff
@@ -357,11 +361,17 @@ fn default_timeout() -> DurationString {
     DurationString::new(Duration::new(30, 0))
 }
 
+fn default_tolerate_bad_tls() -> bool {
+    false
+}
+
```

Comment:
For simple values why don't you use `default_value` instead of defining a function that returns a value?

Reply:
![image](https://github.com/dfinity/metrics-proxy/assets/116065832/87e644fe-c9f0-4e3d-ba00-ce83aa2b6950)
Interesting, bellow is the version:
```
clap = { version = "4.2.7", features = [
    "derive",
    "env",
    "usage",
    "color",
    "help",
    "error-context",
    "suggestions",
    "wrap_help",
    "string",
    "cargo",
] }
```